src/js/structs/DSLFilters.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This class maintains a dictionary with all the currently available
+ */
+class DSLFilters {
+
+  /**
+   * DSLFilters constructor with optional initial filters array
+   *
+   * @param {Array} [filters] - An array with the initial filters
+   */
+  constructor(filters=[]) {
+    this.filters = filters;
+  }
+
+  /**
+   * Plug a filter to the list of filters
+   *
+   * @param {DSLFilter} filter - The filter to plug
+   * @returns {DSLFilters} Returns a DSLFilters object for chaining the calls
+   */
+  plug(filter) {
```

Comment:
Should we throw an error in development mode, if something that is not a DSLFilter is passed?